metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -229,13 +229,20 @@ def _convert_value(param: parameters.Parameter):
         if kwargs.get(param.name) is not None
     }
 
+    # Add additional tags
+    if kfp_namespace:
+        tags = tags + (f"kfp_namespace:{kfp_namespace}",)
+
+    if experiment_name:
+        tags = tags + (f"experiment:{experiment_name}",)
```

Comment:
Or is this the intended usage? (this seems cleaner than looking for the prefix `experiment_`)
This would also create a KFP experiment.

```bash
python hello.py kfp run --experiment foo
```

```python
Flow("Foo").runs("foo")
```

Reply:
yes exactly that's what we discussed. That it is a good idea to add experiment and namespace as tags by default.